tests/models.py

```diff
@@ -57,6 +57,29 @@ def __str__(self):
         return self.name
 
 
+class BaseFood(models.Model):
+    name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class MultiInheritanceLazyResolutionFoodTag(TaggedItemBase):
+    content_object = models.ForeignKey(
+        "MultiInheritanceFood", related_name="tagged_items", on_delete=models.CASCADE
+    )
+
+    class Meta:
+        unique_together = [["content_object", "tag"]]
```

Comment:
instead of unique_together you could try UniqueContraint

Reply:
https://docs.djangoproject.com/en/2.2/ref/models/options/#unique-together

Reply:
There are about a half dozen occurrences of unique_together in this file, and none of them have been switched over to UniqueConstraint yet. I think for clarity, as this would be an unrelated change, it would be better to change all the unique-together to UniqueConstraint with its own issue / PR.

Reply:
UniqueConstraint was introduced in Django 2.2, so if I'm not mistaken, this would also mean the min version of Django would have to be bumped to 2.2 and version 1 support would have to be dropped. This is probably fine at this point, but it is also beyond the scope of this PR.